### ISSUE TITLE
34361 men, women and children who perished trying to reach Europe

### ISSUE THREAD

**Issue by hakantan** [jsoma/data-studio-projects#208]
# Pitch

[The Guardian](https://www.theguardian.com/world/2018/jun/20/the-list-europe-migrant-bodycount) has recently published a story based on a dataset provided to them by the Dutch NGO United for Intercultural Action. The list is "a record of every reported instance in which someone has died trying to migrate into Europe." The Guardian-story is powerfully told, I would like to use the dataset

## Summary
I will look into the dataset and see what can be said about where these people came from, how many men/women/children are among the dead. 

## Details

**Possible headline(s):** 
More than 30 000 people have died trying to come to Europe

**Data set(s):** 
Published by the Guardian [over here](by the Dutch NGO United for Intercultural Action), csv-ready version [over here](https://github.com/ondata/the-list). I have used [Tabula](https://tabula.technology/) for converting the files from PDF to CSV.

**Code repository:** 
[Link](https://github.com/hakantan/data_studio/tree/master/code/03-the_list)
	
**Possible problems/fears/questions:** 

## Work so far

I have spent some time cleaning the dataset a little bit. (Had a tokenization error in pandas.)

So far I have just one tiny picture, with no real analysis behind it. Mainly this is due to the fact that I wanted to have a different project as the third project, but that didn't work out, so here we are. 

<img width="475" alt="bildschirmfoto 2018-07-24 um 00 32 46" src="https://user-images.githubusercontent.com/32862264/43117105-1e8149ec-8ed9-11e8-8b2e-5d171f7c3aa5.png">

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc (not much, tho!)
- [x] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by jsoma** [comment]
I think the story isn't going to be, "here's a magic overall trend," but instead, "let me build a completely impersonal graphic, which I will then use to highlight specific people/cases in history/etc." Think of this as a _story_ or a source of research first instead of a graphic and I think you'll come out a lot better than if you just try to visualize the data.

**Comment by troboukis** [comment]
You need to regex the cause of death. You'll get very important data from there like 'fight', 'killed in fight', 'malnutrition'. The cause of death will highlight several other issues that are under reported regarding the refugee crisis.

**Comment by vpenney** [comment]
I agree that it would be interesting to regex the cause of death. There could be some interesting insights in there as well--I think a lot of people talk about migrants drowning, but maybe there are some other trends you can uncover.

I don't necessarily think that you can infer whether or not people are related in that dataset (except in the case of the two twins that I saw), but you could also take a look at whether women and children tended to die together and things like that. 

I like you're topic and you have a good dataset, but it'll take a little work to sift through it to find your story. I'm looking forward to seeing what you find!

**Comment by Katerinavts** [comment]
I agree with the comments above. It is a massive dataset and I wouldn't just visualize it as a whole. I would say though that this is a story that helps you showcase your visualization skills;  especially the element of surprise. To throw in my two cents, I would try to connect the reason of death with the time/date and location and look for stories there. For example, when we hear about deaths we think of boats that capsized and people who drowned. But suicide incidents are not rare any more either.

**Comment by sarahslo** [comment]
i think i have the same response to this as soma, but think of it a little differently.

so many questions waiting to be answered. are these men? women? children? how old? as with other crisis do the old and the young die most? what are these people dying of? when do they die? is there a season or a pattern to the deaths based on what is going on during an airstrike or a military action? 

can you sort through one of the countries and find out some of these answers and then use it as a model for the other countries?

also, i would drop the 'other' category because it tells me nothing.

**Comment by hakantan** [comment]
# Update
<img width="789" alt="bildschirmfoto 2018-07-31 um 23 32 33" src="https://user-images.githubusercontent.com/32862264/43499595-5a1a1e74-951a-11e8-8572-6df92a7622b9.png">
  
## Any changes in direction or topic?

No changes so far. I'm still sorting through the data. I do think it is interesting to focus on themes besides drowning. There seem to be a lot of suicides, for one. I will look into that further. This picture has no real-styling, I'm just experimenting at this point. Hopefully will include another update by tomorrow.
	
## Problems/Questions
  
The dataset is huge, but needs attention and care. So I spend most of my time reading through the rows, trying to capture all the details. Hence I do not have many graphics to show at this point.
	
## Checklist
	
- [x] I have included ~~my~~ some visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by angelareplica** [comment]
Super interesting project; I agree with the feedback above. If dates are included in the dataset, it would be interesting see whether there are any trends in causes of death over time. E.g. Did more children die in a certain year than others, relative to number of migrants? Are more migrants committing suicide?

On your most recent chart, I'm confused by what a "stowaway" death is. Is that completely distinct from the "suffocated" bar? Seems like there might be some overlap there. 

Looking forward to seeing the final product!

**Comment by adrianblanco** [comment]
Good job! 
+1 to analyze the data over time if dates are included.
Also, I'd consider to include a title in your chart. It is going to be easy to understand.
Finally, as you mentioned, suicides can be an interesting topic which is not really covered in immigration topics! Looking forward to new updates!
